angular/directives/README.md

```diff
@@ -0,0 +1,101 @@
+# Directives
+
+Angular directives is custom field in HTML. It will add custom logic to HTML elements. For example, changing CSS or HTML contents.
+
+## Creating Directives Manually
+
+Step below will guide you by example.
+
+### Goal
+
+You want to create custom field `appHighlight` which will add background color in the HTML element.
```

Comment:
Can we add the instruction about naming a new directive krub? For example, we should prefix them with `app` (need to followed from `angular.json`).